src/usePerpsGetMarketSummary.ts

```diff
@@ -0,0 +1,58 @@
+import { useQuery } from '@tanstack/react-query';
+import type { ethers } from 'ethers';
+import { fetchPerpsGetMarketSummary } from './fetchPerpsGetMarketSummary';
+import { fetchPerpsGetMarketSummaryWithPriceUpdate } from './fetchPerpsGetMarketSummaryWithPriceUpdate';
+import { useImportContract } from './useImports';
+import { usePriceUpdateTxn } from './usePriceUpdateTxn';
+import { useSynthetix } from './useSynthetix';
+
+interface PerpsGetMarketSummary {
+  skew: ethers.BigNumber;
+  size: ethers.BigNumber;
+  maxOpenInterest: ethers.BigNumber;
+  currentFundingRate: ethers.BigNumber;
+  currentFundingVelocity: ethers.BigNumber;
+  indexPrice: ethers.BigNumber;
+}
+
```

Comment:
```suggestion
```

Reply:
Interface removed.

Moved return type description to functions `fetchPerpsGetMarketSummary` and `fetchPerpsGetMarketSummaryWithPriceUpdate`.